### PR TITLE
Add description across SLSA pages.

### PR DESCRIPTION
**Pull request by arewm**
Link unfurling and SEO currently only uses the default description for the entire webpage. In order to provide more clarity to this, I unified pages' frontmatter to specify the title and provide a description.

Resolves: #693

Blog posts don't need to have descriptions added as they seem to be handled differently. Their description is populated by the initial content in the post.

I added basic one-liner content to each of the pages, but the HTML tag (and unfurling) can handle longer descriptions. Feel free to offer suggestions about what the description should be.

I also modified the earlier versions (i.e. 1.0-rc draft) as I figured that this was beneficial and does not directly affect the content displayed in the specifications.

### CODE REVIEW

**Comment by arewm** [comment]
A preview of the difference:
Screenshot 2023-03-16 at 11 03 57 AM

**Review by lehors — APPROVED** [review]
LGTM.

**Comment by arewm** [comment]
@lehors, I rebased on a recent commit to deconflict, but I also inflated the descriptions in the v1.0 directory as I since realized that it is okay for the snippets to be more than just a simple short line.

@olivekl, FYI as well as I moved the first headings into the frontmatter as a title to ensure that it is picked up by the SEO plugin.

File: docs/attestation-model.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Software attestations
+description: Overview of software attestations and is specifics related to SLSA.
```
Comment (joshuagl) [inline docs/attestation-model.md:3]:
```suggestion
description: Overview of software attestations and its specifics related to SLSA.
```
?

**Review by joshuagl — CHANGES_REQUESTED** [review]
This is a great set of changes, thank you! Two high-level questions/suggestions:
1. can we finish each description with a period (.) for consistency?
2. should we include the version in descriptions, so that it's easy to tell, i.e., v0.1 requirements form v1.0 requirements in a preview?

**Review by joshuagl — APPROVED** [review]
Thanks for the fixes Andrew. Your rationale for not including the versions seems sane to me, let's not do anything with that suggestion for now.

**Comment by lehors** [comment]
We're better off not adding the version to this because it would be one more thing we'd have to maintain. We need to try and limit the number of things that need updating every time we produce a new release.

**Review by trishankatdatadog — APPROVED** [review]

**Pull request merged**